database/migrations/2023_01_09_134359_add_event_column_to_activity_log_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+final class AddEventColumnToActivityLogTable extends Migration
+{
+    public function up()
+    {
+        Schema::connection(config('activitylog.database_connection'))->table(config('activitylog.table_name'), function (Blueprint $table) {
+            $table->string('event')->nullable()->after('subject_type');
```

Comment:
Is this not provided in the migration stub that creates the `activity_log` table? If not, is this column supported by out of the box by the activity log library? Because I don't seem to see any custom code that stores data in this extra column?

Reply:
I went through the migrations with Jonas and I proposed to create one simple migration that also uses the table name.

Reply:
Ah I found it, it's a separate migration in the package itself https://github.com/spatie/laravel-activitylog/blob/main/database/migrations/add_event_column_to_activity_log_table.php.stub

That was not clear to me.